Packs/Code42/Integrations/Code42/integration-Code42.yml

```diff
@@ -0,0 +1,1037 @@
+commonfields:
```

Comment:
This file is auto-generated. I am only including because you basically load this into any Cortex XSOAR server and use our integration, so if QA wants to test our integration, they don't have to use the `demisto-sdk` to generate this file.